packages/frontend/src/style.scss

```diff
@@ -72,7 +72,8 @@ html {
 	}
 
 	&.useSystemFont {
-		font-family: 'Hiragino Maru Gothic Pro', sans-serif;
+		// https://css-tricks.com/snippets/css/system-font-stack/#:~:text=/*%20System%20Fonts%20with%20system%2Dui%20*/
+		font-family: system-ui, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
https://caniuse.com/font-family-system-ui

Sounds like `system-ui` alone is enough, as Misskey does not support older browsers.

Reply:
Ah, great! That simplifies things then.

Done.